interfaces/IWear/include/Wearable/IWear/Common.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef WEARABLE_COMMON_H
+#define WEARABLE_COMMON_H
+
+#ifndef wError
+#include <iostream>
+#define wError std::cerr
+#endif
+
+#ifndef wWarning
+#include <iostream>
+#define wWarning std::cout
+#endif
+
+#include <string>
+
+namespace wearable {
+
+    const std::string Separator = "::";
+
+    enum class DeviceType
```

Comment:
Instead of `DeviceType` may be `ElementType` will be a cleaner naming, where an element is either a sensor or an actuator.

Reply:
Done in https://github.com/robotology/wearables/pull/105/commits/ec5c57d1e4aaa41a97e520ceaa10824159b95b80